README.md

```diff
@@ -58,6 +58,7 @@ Note:
    takes precedence, otherwise the API key is used. If none of these three parameters are passed, then preference will be given to the
    domino token file from the corresponding environment variable, then to the API key from the corresponding environment variable.
 2. By default the log level is set to `INFO`, to set log level to `DEBUG`, set `DOMINO_LOG_LEVEL` environment variable to `DEBUG`
+3. For testing purposes and issues with ssl certificates, set environment variable `DOMINO_NOT_VERIFY_CERT` to `true`. Be sure to unset this variable when not in use.
```

Comment:
I usually try to use the positive form for my boolean variable or parameter names to avoid double-negative logic (`if not notGood`). 

I think the env var here should be called `DOMINO_VERIFY_CERTIFICATE` and it should default to `True`.

Reply:
But it's not a deal-breaker, either.